### PR TITLE
Validate sle repos more slowly and remove repo check on source system

### PR DESCRIPTION
**Pull request by mitiao**
See: https://progress.opensuse.org/issues/19540

### CODE REVIEW

File: products/sle/main.pm [hunk 1]
```diff
@@ -993,7 +993,6 @@ sub load_online_migration_tests() {
     # stop packagekit service and more
     loadtest "online_migration/sle12_online_migration/online_migration_setup";
     loadtest "online_migration/sle12_online_migration/register_system";
-    loadtest "online_migration/sle12_online_migration/repos_check";
```
Comment (okurz) [inline products/sle/main.pm]: This test module does not seem to used anywhere now, delete? Also I think this is an unrelated change.

Comment (mitiao) [inline products/sle/main.pm]: deleted.

Comment (mitiao) [inline products/sle/main.pm]: Split to 2 commits for different purposes.
